resources/monitoring/charts/grafana/dashboards/kubernetes-apiserver-dashboard.json

```diff
@@ -1,859 +0,0 @@
-{
```

Comment:
I see new apiserver dashboards but I cannot find this one with general http metrics, is there a replacement for it?

Reply:
there is some changes in the original chart, if we still need old dashboard we can add as a custom chart to the kyma

Reply:
I added the old chart